www/templates/components/trades/payout.template.html

```diff
@@ -16,7 +16,7 @@
     <div class="col col-33 basis" ng-show="basis == 'payout'">{{'trade.payout' | translate}}</div>
     <div class="col col-33 basis" ng-show="basis == 'stake'">{{'trade.stake' | translate}}</div>
     <div class="col col-33 amount">
-        <input type="number" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
+        <input type="text" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
```

Comment:
I have known the problem but unfortunately when we set input type to number
HTML engine doesn't accept anything else numbers and digits so when you
enter '1.' the engine ignore this value and we don't have access to it in
angular. So I cant find that what is the exact value in the input box.

On Sat, Feb 27, 2016 at 6:45 PM, Ashkan <notifications@github.com> wrote:

> In www/templates/components/trades/payout.template.html
> <https://github.com/binary-com/mobile/pull/317#discussion_r54334594>:
>
> > @@ -16,7 +16,7 @@
> >      <div class="col col-33 basis" ng-show="basis == 'payout'">{{'trade.payout' | translate}}</div>
> >      <div class="col col-33 basis" ng-show="basis == 'stake'">{{'trade.stake' | translate}}</div>
> >      <div class="col col-33 amount">
> > -        <input type="number" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
> > +        <input type="text" string-to-number name="amount" ng-model="amount" ng-change="updateAmount(amount, '{{amount}}')" required step="0.01">
>
> Problem is when user enters "1." , we sent "amount":0, so server return
> error.
> When user enters "1.", we should send "amount":1, then it works as
> expected.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/mobile/pull/317/files#r54334594>.
>